doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -646,7 +646,14 @@ Scripts
 
 This section provides detailed lists of changes by :ref:`script <scripts>`.
 
-* Added semantic version support to :ref:`nrf_desktop_config_channel_script` Python script for devices that use the SUIT DFU.
```

Comment:
What happened here? @divipillai 

This entry was already mentioned in NCS 2.8 as I added it:

https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/releases_and_maturity/releases/release-notes-2.8.0.html#scripts

Reply:
That was in 2.8.0, so I guess we do not need that entry for 2.9.0. Please add the new stuff as a comment to my https://github.com/nrfconnect/sdk-nrf/pull/19240 and remove from this PR.

Reply:
I will fix this (in separate PR). I will consult with @divipillai

Reply:
My bad, I missed to clean this entry while cleaning the changelog. Checked the Change log cleanup PR again and this is the only entry I missed :) . Good catch. Since we are already removing the entry in this PR, I think no action is required here.

Reply:
Some mishap during the releasing I guess. It was the first time we had the preview doc release and releasing from branch.

Reply:
Thanks for the explanation. We are not removing this entry in this PR but @MarekPieta promised to do that in a follow-up PR. @MarekPieta, if you plan to update this PR, you can consider adding a separate commit that remove this outdated entry

Reply:
Eventually will be removed as part of @peknis PR: https://github.com/nrfconnect/sdk-nrf/pull/19240#discussion_r1869608845?

Reply:
@MarekPieta, the latest changelog is not changed in @peknis PR

Reply:
oh I think the changelog reset will happen as part of the following PR:

https://github.com/nrfconnect/sdk-nrf/pull/19241

Reply:
Due to the releasing-from-branch approach, there will be two resets. In both 19241 and 19240. Complicated and new to me as well. You better discuss with Divya as she has a bit of experience already.

Reply:
Yeah, your understanding here is correct @kapi-no .

The Changelog will be rest in https://github.com/nrfconnect/sdk-nrf/pull/19241 and the new entry for the 2.9 release could be added to https://github.com/nrfconnect/sdk-nrf/pull/19240 , which is already done. I think its sorted now.